src/ansys/mapdl/core/xpl.py

```diff
@@ -290,13 +308,35 @@ def save(self):
         self._check_ignored(response)
         return response
 
+    def extract(self, recordname, sets="ALL", asarray=False):
```

Comment:
New method on the XPL command. This allows the reading of a matrix of any MADPL file, once it's visible through the XPL listing.